### DISCUSSION TITLE
*Is there a way to make the event longer?*

### DISCUSSION THREAD

**Discussion by AlexandertheIght** [TEHE-Studios/ExpandedHelicopterEvents#192] [Q&A] [answered]
I feel like the helicopter events are very short which I like and for the most part makes sense but is there a way to have the military last longer like if they were on a multi-hour mission. Is there a way to expand its timeframe? Also, will the military ineventbley kill me or is there a way to stop it?

**Comment by AlexandertheIght** [comment]
I only did  * it because it told me too I know nothing of this platform other then it is intended to code just made this to ask this

**Comment by Chuckleberry-Finn** [comment, marked as answer]
No there is no sandbox option to make duration of individual events longer. There is a behavior involving military events to avoid their wrath, and that is to wear military clothing.
